docs/core/tools/dotnet-tool-uninstall.md

```diff
@@ -1,62 +1,73 @@
 ---
 title: dotnet tool uninstall command
-description: The dotnet tool uninstall command uninstalls the specified .NET Core Global Tool from your machine.
-ms.date: 05/29/2018
+description: The dotnet tool uninstall command uninstalls the specified .NET Core tool from your machine.
+ms.date: 02/14/2020
 ---
 # dotnet tool uninstall
 
-[!INCLUDE [topic-appliesto-net-core-21plus.md](../../../includes/topic-appliesto-net-core-21plus.md)]
+**This article applies to:** ✔️ .NET Core 2.1 SDK and later versions
 
 ## Name
 
-`dotnet tool uninstall` - Uninstalls the specified [.NET Core Global Tool](global-tools.md) from your machine.
+`dotnet tool uninstall` - Uninstalls the specified [.NET Core tool](global-tools.md) from your machine.
 
 ## Synopsis
 
 ```dotnetcli
 dotnet tool uninstall <PACKAGE_NAME> <-g|--global>
 dotnet tool uninstall <PACKAGE_NAME> <--tool-path>
+dotnet tool uninstall <PACKAGE_NAME>
 dotnet tool uninstall <-h|--help>
 ```
 
 ## Description
 
-The `dotnet tool uninstall` command provides a way for you to uninstall .NET Core Global Tools from your machine. To use the command, you either have to specify that you want to remove a user-wide tool using the `--global` option or specify a path to where the tool is installed using the `--tool-path` option.
+The `dotnet tool uninstall` command provides a way for you to uninstall .NET Core tools from your machine.  To use the command, you specify one of the following:
```

Comment:
Double space again. Comment from previous section applies here. The same point I made about the bullet points in the `dotnet tool install` command.

```suggestion
The `dotnet tool uninstall` command provides a way for you to uninstall .NET Core tools from your machine. To use the command, you specify one of the following:
```